day_7/checker_board.rb

```diff
@@ -0,0 +1,11 @@
+#checker board
```

Comment:
Good job on this! I encourage you to attempt the rest of the Day-7 exercises to keep building those ruby muscles!